sequencing-server/src/defaultSeqBuilder.ts

```diff
@@ -42,3 +44,43 @@ export const defaultSeqBuilder: SeqBuilder = (
     steps: commands,
   });
 };
+
+// Order commands by time in expanded sequences.
+function sortCommandsByTime(commands: CommandStem<{} | []>[]): CommandStem<{} | []>[] {
+  const relativeTimeTracker: Record<string, Temporal.Instant> = {};
+  let previousAbsoluteTimestamp: Temporal.Instant | null = null;
+
+  for (const command of commands) {
+    // If the command is epoch-relative, complete, or the first command and relative then short circuit and don't try and sort.
+    if (
+      command.epochTime !== null ||
+      (command.absoluteTime === null && command.epochTime === null && command.relativeTime === null) ||
```

Comment:
Are these explicitly set to `null` if they do not exist?

Reply:
Yeah!
https://github.com/NASA-AMMOS/aerie/blob/feature/time-order-commands/sequencing-server/src/lib/codegen/CommandEDSLPreface.ts#L614-L616